platform-common-resources/modules.tf

```diff
@@ -38,4 +38,8 @@ module "cert-manager" {
   certificate_key_content  = var.certificate_key_content
 
   depends_on = [module.create-ingress-nginx]
+}
+
+module "loki" {
+  source = "./create-loki"
```

Comment:
we need to add all references of variables if somebody needs changes default values

1. monitoring_namespace
2. namespace
3. loki_release_name
4. loki_persistence_memory
5. loki_retention_period
6. helm_repo_url
7. helm_chart
8. loki_max_entries_limet_per_query

Reply:
Hello, I added the references to these variables, as asked.

Reply:
Hello, thank you for that. Now i see that we need to add all references of variables in parent module (create-prometheus-stack) and parent of parent module. Do you need some help for this task ?